lsp/nls/src/command.rs

```diff
@@ -26,12 +23,13 @@ pub fn handle_command(
 }
 
 fn eval(server: &mut Server, uri: &Url) -> Result<(), Error> {
-    if let Some(file_id) = server.cache.file_id(uri)? {
+    let cache = &mut server.world.cache;
```

Comment:
Nitpick: it seems we only cache once below, and not as a ref (we call `clone()`), so we might just inline its definition as well.